src/main/java/com/microsoft/spring/data/gremlin/annotation/Edge.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.annotation;
```

Comment:
so package is com.microsoft.spring.xxx or com.microsoft.azure.spring.xxx? seems first one more general

Reply:
if use first one, pls update pom file to be consistent